docs/stacks-guide-aws.md

```diff
@@ -9,144 +9,163 @@ indent: true
-`default`. In this guide, we assume that the credentials are configured
-under the `crossplane-user` profile, but you can use a different profile
-name if you want. Let's store the profile name in a variable so we can
-use it in later steps:
+It should look something like:
 
 ```bash
-export aws_profile=crossplane-user
+NAME        READY   VERSION   AGE
+stack-aws   True    0.0.2     45s
 ```
 
+## Configure the AWS account
+
+An AWS [user][aws user] with `Administrative` privileges is needed to enable
+Crossplane to create the required resources. Once the user is provisioned, an
+[Access Key][] needs to be created so the user can have API access.
+
+Using the set of [access key credentials][AWS security credentials] for the user
+with the right access, we need to [install][install-aws] [`aws cli`][aws command
+line tool], and then [configure][aws-cli-configure] it.
+
+When the AWS cli is configured, the credentials and configuration will be in
+`~/.aws/credentials` and `~/.aws/config` respectively. These will be consumed in
+the next step.
+
+When configuring the AWS cli, the user credentials could be configured under a
+specific [AWS named profile][], or under `default`. Without loss of generality,
+in this guide let's assume that the credentials are configured under the
+`aws_profile` profile (which could also be `default`). We'll use this profile to
+setup cloud provider in the next section.
+
 ## Configure Crossplane Provider for AWS
 
-Crossplane uses the aws user credentials that were configured in the previous
+Crossplane uses the AWS user credentials that were configured in the previous
 step to create resources in AWS. These credentials will be stored as a
-[secret][] in Kubernetes, and will be used by an [aws
-provider][aws-provider-docs] instance. The AWS region is also pulled
-from the cli configuration, so that the aws provider can target a
-specific region.
+[secret][kubernetes secret] in Kubernetes, and will be used by an [AWS
+provider][aws provider] instance. The default AWS region is also pulled from the
+cli configuration, and added to the AWS provider.
 
 To store the credentials as a secret, run:
 
 ```bash
 # retrieve profile's credentials, save it under 'default' profile, and base64 encode it
-export AWS_CREDS_BASE64=$(echo -e "[default]\naws_access_key_id = $(aws configure get aws_access_key_id --profile $aws_profile)\naws_secret_access_key = $(aws configure get aws_secret_access_key --profile $aws_profile)" | base64  | tr -d "\n")
+BASE64ENCODED_AWS_ACCOUNT_CREDS=$(echo -e "[default]\naws_access_key_id = $(aws configure get aws_access_key_id --profile $aws_profile)\naws_secret_access_key = $(aws configure get aws_secret_access_key --profile $aws_profile)" | base64  | tr -d "\n")
 # retrieve the profile's region from config
-export AWS_REGION=$(aws configure get region --profile ${aws_profile})
+AWS_REGION=$(aws configure get region --profile ${aws_profile})
```

Comment:
if run as separate commands from the bash script below, think we still need `export`

Reply:
see my other comment